dirjump

```diff
@@ -117,6 +150,7 @@ dirjump()
 delete_a_dir_path()
 {
 	grep -Fxv "$1" "$directory_list" > temp; mv temp "$directory_list"
+	date +%s > "$last_dir_remove"
```

Comment:
This line should be moved to somewhere else. The function `delete_a_dir_path` was originally purposed to [prevent duplicate paths](https://github.com/AutomationBoi/dirjump/blob/ff74ae408a3ee0600c5cc03ff0b5117ab52ed7e2/dirjump#L164) when a  path (already in the list) is added to the top of the list. A path is really removed from the directory history when the list is full and a new path (not in the list yet) is added to the top of the list.

Reply:
![image](https://user-images.githubusercontent.com/34421249/66616003-3cbe4f80-ebf9-11e9-9cae-d718ef5ada0f.png)

In the above image, it says a path was last removed just now, but actually nothing was removed. It just moves `~/.config` from 4 to 1.